packages/terra-action-footer/tests/jest/ActionFooter.test.jsx

```diff
@@ -91,6 +96,14 @@ describe('ActionFooter', () => {
         expect(actionFooter).toMatchSnapshot()
       ));
     });
+
+    describe('with no border', () => {
```

Comment:
I think we can drop one of these jest test. Seems to be testing the same thing

Reply:
With that being said, should have matching tests ActionFooter and CenterActionFooter.